dandischema/models.py

```diff
@@ -550,14 +563,26 @@ class AccessRequirements(DandiBaseModel):
     embargoedUntil: Optional[date] = Field(
         None,
         title="Embargo end date",
-        description="Date on which embargo ends",
+        description="Date on which embargo ends.",
         readOnly=True,
         nskey="dandi",
         rangeIncludes="schema:Date",
     )
 
     _ldmeta = {"rdfs:subClassOf": ["schema:Thing", "prov:Entity"], "nskey": "dandi"}
 
+    """
+    # TODO: Enable when embargoed access is supported.
+    @root_validator
+    def open_or_embargoed(cls, values):
+        status, embargoed = values.get("status"), values.get("embargoedUntil")
+        if status == AccessType.EmbargoedAccess and embargoed is None:
+            raise ValueError("An embargo date. For NIH supported awards, this "
+            "must comply with NIH policy. For other datasets, this will be at "
+            "most a year from dandiset creation.")
```

Comment:
```suggestion
            "most a year from the dandiset creation.")
```
since of a specific dandiset creation time.